chain/jsonrpc/tests/rpc_query.rs

```diff
@@ -16,7 +16,18 @@ fn test_block() {
 
         let mut client = new_client(&format!("http://{}", addr));
         actix::spawn(client.block(0).then(|res| {
-            assert_eq!(res.unwrap().header.height, 0);
+            let res = res.unwrap();
+            assert_eq!(res.header.height, 0);
+            assert!(res.header.epoch_hash.0.len() > 0);
```

Comment:
Why not assert the exact length?

Reply:
even better, why not assert the value? this is genesis block, values going to be the same except timestamp.

Reply:
I guess hash itself will be different due to timestamp as well.

@bowenwang1996 exact length though due to bs58 can be different depending on value.

Reply:
Yes but we can decode and then assert the length